src/oom/oomd-util.c

```diff
@@ -122,6 +122,16 @@ uint64_t oomd_pgscan_rate(const OomdCGroupContext *c) {
         return c->pgscan - last_pgscan;
 }
 
+bool oomd_mem_free_below(const OomdSystemContext *ctx, int threshold_permyriad) {
+        uint64_t mem_threshold;
+
+        assert(ctx);
+        assert(threshold_permyriad <= 10000);
+
+        mem_threshold = ctx->mem_total * threshold_permyriad / (uint64_t) 10000;
```

Comment:
to be fully correct regarding overflow, maybe:

```c
if (ctx->mem_total <= UINT64_MAX/10000)
        mem_threshold = ctx->mem_total * threshold_permyriad / (uint64_t) 10000;
else
        mem_threshold = ctx->mem_total / (uint64_t) 10000 * threshold_permyriad;
```

(but we don't do this elsewhere, so doesn't matter